src/md/src/UserGuide/Post bug to Bug Tracking System.md

```diff
@@ -1,52 +1,69 @@
 Post bug to Bug Tracking System
 --------------------------------
 
-Report Portal allows to connect JIRA, TFS and RALLY and post bug to them.
+Report Portal allows to connect JIRA or RALLY systems and post bug to them.
 
 Before post a bug, make sure that the bug tracking systems is connected to the
 project on the project settings page.
 
 In case bug tracking system is not connected, "Post bug" button will be
 disabled.
 
+The functionality is available for all users and does not depends on their project role.
 
 ### Post bug to JIRA
 
-In order to post bug to JIRA, perform the following steps:
+In order to post a bug to JIRA, perform the following steps:
 
 1. Navigate to the "Launches" page.
 
 2. Drill down to the Step or Log level of any failed item.
 
-3. Click "Defect editor" link or pencil icon in the "Defect Type" column.
+3. Click "Post Bug" button.
 
-4. [ ![Image](Images/pic_110.jpg) ](Images/pic_110.jpg)
-
-5. You will see the defect editor block. Click "Post Bug" icon from the
-    editor header.
-
-6. [ ![Image](Images/pic_115.jpg) ](Images/pic_115.jpg)
+4. Post bug pop-up form appears with selected set of fields: required (with asterisk mark) and optional; 
+include data and credentials
 
-7. Post bug pop-up form should appear.
+5. Fill in the fields with correct information
 
-8. [ ![Image](Images/pic_116.jpg) ](Images/pic_116.jpg)
+6. Select include data options or leave the default.
 
-9. Select JIRA project, fill in the form and click "Submit" button.
+7. Fill in credentials of your JIRA account and click 'Post' button
 
-10. The bug will be posted to the JIRA and link to the bug will be displayed on the
+8. The bug will be posted to the JIRA and link to the bug will be displayed on the
 Step and Log levels of the failed item.
 
-Hover the mouse over the bug and you will see issue status and summary on
+9. Hover the mouse over the bug and you will see issue status and summary on
 tool-tip.
+ 
+10. To view data of just submitted bug in JIRA, click the ticket. All selected data is present in bug tracking system.
+Including logs, attachments and comments.
 
-[ ![Image](Images/pic_117.jpg) ](Images/pic_117.jpg)
+11. To get to item in Report Portal for which the bug was posted, click back link 'Link to defect'. 
 
-**Note:**
-Project name in JIRA on post bug pop-up will be missing, if only one JIRA project is configured on the current project.
+Sometimes it necessary to add one bug to some items via bulk operation. To do this:
 
-Also you can post bug on Log level.
+1. Go to Step level of the launch
+
+2. Select some failed items by click on their checkboxes
+
+3. Expand 'Actions' list on the top of the page
+
+4. Click 'Post Bug' option
+
+5. Fill in the fields with correct information
+
+6. Fill in credentials of your JIRA account and click 'Post' button
+
+7. Ticket of bug is added to all selected items.
```

Comment:
Here's the alternative to point#7:
One ticket with bug is added to JIRA as for single bug posting. The identifier of JIRA ticket is added to all selected items. Keep in mind that data of Report Portal items are excluded in a bulk posting.